test/examples/make.jl

```diff
@@ -20,7 +20,7 @@ else
     ["html", "html-meta-custom", "html-mathjax2-custom", "html-mathjax3", "html-mathjax3-custom",
     "html-local", "html-draft", "html-repo-git", "html-repo-nothing", "html-repo-error",
     "html-sizethreshold-defaults-fail", "html-sizethreshold-success", "html-sizethreshold-ignore-success", "html-sizethreshold-override-fail", "html-sizethreshold-ignore-success", "html-sizethreshold-ignore-fail",
-    "latex_texonly", "latex_simple_texonly", "latex_showcase_texonly", "html-pagesonly"]
```

Comment:
We don't want to run the full LaTeX builds as part of the main test suite (because of the dependencies). But we have a special test file for running those:

https://github.com/JuliaDocs/Documenter.jl/blob/7251be3e45579591f755b668e8e364b367f5b7a6/test/examples/tests_latex.jl#L3-L7

Which we run as part of this workflow:

https://github.com/JuliaDocs/Documenter.jl/blob/7251be3e45579591f755b668e8e364b367f5b7a6/.github/workflows/CI.yml#L52-L83

Reply:
This was reverted